Automattic/wc-calypso-bridge
# Patch wc.data for Gutenberg 15.5+ and wc <= 7.6.0

### Changes proposed in this Pull Request:



Closes # .






### How to test the changes in this Pull Request:




1. Remove Navigation disable code

- https://github.com/Automattic/wc-calypso-bridge/blob/f7cef0091e4c06d95cb009e60c17da4cf29cda3c/includes/class-wc-calypso-bridge-woocommerce-admin-features.php#L93-L95
- https://github.com/Automattic/wc-calypso-bridge/blob/f7cef0091e4c06d95cb009e60c17da4cf29cda3c/includes/class-wc-calypso-bridge-woocommerce-admin-features.php#L136-L137

4.Go to WooCommerce -> Settings -> Features
5. Enable the Navigation feature:
- Undo changes from this [PR](https://github.com/woocommerce/woocommerce/pull/36456/files) to enable the new navigation option
- Or run `wp option set woocommerce_navigation_enabled yes`
6. Confirm there are errors `peError: object null is not iterable (cannot read property Symbol(Symbol.iterator))` in dev console
7. Apply this patch
8. Confirm the no errors in the dev console
9. You might still not see the navigation menu in /wp-admin/admin.php?page=wc-settings&tab=advanced I'm not really sure why, but I think it's something related to bridge/atomic code
10. Go to wp-admin/admin.php?page=wc-admin 
11. Confirm navigation works



### Other information:




### FOR PR REVIEWER ONLY:
